backend/app/services/transcription.py

```
@@ -0,0 +1,283 @@
+        self._in_speech = False
+        return [finalized]
+
+    async def _finalize_segment(
+        self,
+        pcm16_audio: bytes,
+        start_ts_ms: int,
+        end_ts_ms: int,
+        session_id: str,
+        day_id: str,
+    ) -> TranscriptSegment:
+        text, confidence, retries = await self._transcribe_with_retry(pcm16_audio)
```

Ruff (0.15.9)

[warning] Unpacked variable `retries` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)